base/abstractarraymath.jl

```diff
@@ -406,3 +406,36 @@ _reperr(s, n, N) = throw(ArgumentError("number of " * s * " repetitions " *
 
     return R
 end
+
+"""
+    eachrow(A::AbstractVecOrMat)
+
+Creates a generator that iterates over the first dimension of matrix A, returning the rows as views.
```

Comment:
```suggestion
Creates a generator that iterates over the first dimension of vector or matrix `A`,
returning the rows as views.
```
(same below, in particular for missing backticks)

Reply:
Missed what you meant about the backticks, but I don't think this is worth running CI again over.

Reply:
I mean that there are more occurrences of code or object names that need backticks below. You can put `[ci skip]` in the commit message to avoid running CI on it.